vipps-qr-api.md

```diff
@@ -313,3 +313,105 @@ does what one might expect, it deletes the QR. Once deleted, merchants can gener
 In addition to the `DELETE`-endpoint, it is also possible to add a `ttl`-attribute in the original `POST`-request. This attribute sets how many seconds the QR will live, before it is deleted permanently.
 
 Tip: If you want the same QR in different formats, perform `GET` calls on the same `id` with different `accept` headers and test what works best.
+
+## Merchant Callback QR codes
+
+Merchant callback QRs makes it possible for users to notify merchants that they want to pay with Vipps. It is the right solution for self-checkout, vending machines, or similar situations where there is no cashier, buttons, or other ways of letting the user communicate that they want to pay with Vipps.
+
+A merchant callback QR is given a `merchantQrId` that the merchant selects upon creation. Then, when the user scans the QR, a callback is sent to the merchant containing this `merchantQrId`. Thus, the merchant knows which QR code has been scanned and they can act accordingly, such as starting a payment towards the user.
```

Comment:
`a callback is sent to the merchant containing this `merchantQrId' **and a 'sessionId'**`

Reply:
The sessionId is relevant in the webhooks documentation for the QrScannedEvent. Not so much here :)

Reply:
Once we add it to webhook we can reference it here